src/utils/consts/consts_vars.go

```diff
@@ -0,0 +1,44 @@
+package consts
+
+import (
+	"fmt"
+
+	style "github.com/NeowayLabs/semantic-release/src/utils/stdout_style"
+)
+
+const (
+	Major                         = "MAJOR"
+	Minor                         = "MINOR"
+	Patch                         = "PATCH"
+	VersionPattern                = "%d.%d.%d"
+	VersionPyVariable             = "__version__"
+	ChangeLogDefaultFile          = "CHANGELOG.md"
+	SetupPythonDefaultFile        = "setup.py"
+	CommitMsgLimitCharacters      = 150
+	AuthorChangelogFormat         = "@%s"
+	ChangeLogCommitHashLinkFormat = "[%s](https://%s/%s/%s/-/commit/%s)"
+)
+
+var (
+	CommitChangeTypes             = []string{"build", "ci", "docs", "fix", "feat", "perf", "refactor", "style", "test", "breaking change", "skip", "skip versioning", "skip v"}
+	CommitChangeTypesMajorUpgrade = []string{"breaking change"}
+	CommitChangeTypesMinorUpgrade = []string{"feat"}
+	CommitChangeTypePatchUpgrade  = []string{"build", "ci", "docs", "fix", "perf", "refactor", "style", "test"}
+	CommitTypeSkipVersioning      = []string{"skip", "skip versioning", "skip v"}
+)
+
+func PrintCommitTypes() {
```

Comment:
@esequielvirtuoso missing `breaking change(s)` to this list

Reply:
Just added.